### PR TITLE
Mark `kubelet` module as beta

### PR DESCRIPTION
**Pull request by exekias**
I also did some small style fixes and added the changelog entry for the module

### CODE REVIEW

File: metricbeat/module/kubelet/container/container.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ type MetricSet struct {
 // Part of new is also setting up the configuration by processing additional
 // configuration entries if needed.
 func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
-	logp.Warn("BETA: The kubelet container metricset is alpha")
+	logp.Warn("BETA: The kubelet container metricset is beta")
```
Comment (ruflin) [inline metricbeat/module/kubelet/container/container.go]: We recently added `logp.Beta(...)` which makes it easier for us to find. Lets switch to this one.

File: metricbeat/module/kubelet/node/node.go [hunk 2]
```diff
@@ -41,7 +41,7 @@ type MetricSet struct {
 // Part of new is also setting up the configuration by processing additional
 // configuration entries if needed.
 func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
-	logp.Warn("BETA: The kubelet node metricset is alpha")
+	logp.Warn("BETA: The kubelet node metricset is beta")
```
Comment (ruflin) [inline metricbeat/module/kubelet/node/node.go]: see comment above.

File: metricbeat/module/kubelet/system/system.go [hunk 3]
```diff
@@ -41,7 +41,7 @@ type MetricSet struct {
 // Part of new is also setting up the configuration by processing additional
 // configuration entries if needed.
 func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
-	logp.Warn("BETA: The kubelet system metricset is alpha")
+	logp.Warn("BETA: The kubelet system metricset is beta")
```
Comment (ruflin) [inline metricbeat/module/kubelet/system/system.go]: same

File: metricbeat/module/kubelet/volume/volume.go [hunk 4]
```diff
@@ -41,7 +41,7 @@ type MetricSet struct {
 // Part of new is also setting up the configuration by processing additional
 // configuration entries if needed.
 func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
-	logp.Warn("BETA: The kubelet volume metricset is alpha")
+	logp.Warn("BETA: The kubelet volume metricset is beta")
```
Comment (ruflin) [inline metricbeat/module/kubelet/volume/volume.go]: same

File: metricbeat/module/kubelet/pod/pod.go [hunk 5]
```diff
@@ -41,7 +41,7 @@ type MetricSet struct {
 // Part of new is also setting up the configuration by processing additional
 // configuration entries if needed.
 func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
-	logp.Warn("BETA: The kubelet pod metricset is alpha")
+	logp.Warn("BETA: The kubelet pod metricset is beta")
```
Comment (ruflin) [inline metricbeat/module/kubelet/pod/pod.go]: same

File: metricbeat/module/kubelet/container/container.go (hunk 1 above)
Comment (exekias) [inline metricbeat/module/kubelet/container/container.go]: Indeed, thanks! updating the PR now

**Comment by ruflin** [comment]
Part of https://github.com/elastic/beats/issues/3965

**Pull request merged**